Shooting/Shooting/Scene/GameScene/Object/GameBackground.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "../../../GameLib/ObjectManager/Object2DBase.h"
+#include "../../../GameLib/DirectX/DirectX.h"
+#include "../../../GameLib/2DHelper/2DHelper.h"
+#include "../../../GameLib/Texture/Texture.h"
+
+
+/**
+* @brief �^�C�g���̔w�i�֌W�̏�����܂Ƃ߂��N���X
+*/
+class GameBackground : public Object2DBase
+{
+public:
+
+	GameBackground();
+	~GameBackground();
+
+	/**
+	* @brief ������֐�
+	* @return ������肭�ł��Ă��Ȃ���FALSE
+	*/
+	bool Initialize();
+
+	/**
+	* @brief ����֐�
+	*/
+	void Release();
+
+	/**
+	* @brief ����֐�
+	*/
+	void Update();
+
+	/**
+	* @brief �`��֐�
+	*/
+	void Render();
+
+private:
+	const char* m_texturePath = "../Graphics/Game_BG.png";
+	const float m_textureTopLeft_x = 128.f;
+	const float m_textureTopLeft_y = 0.0f;
+	const float m_texture_width = 384.f;
```

Comment:
他のメンバ変数はプレフィクス `m_` + LowerCamelとなっているので、### 
```suggestion
	const float m_textureWidth = 384.f;
```
では？

Reply:
関数内の定数にすると同時に命名が変更されていることを確認しました。